test/acceptance/news-redirects.hurl

```diff
@@ -0,0 +1,28 @@
+GET {{host}}/not-found
+HTTP 404 # expect not found response
+[Asserts]
+status == 404
+
+GET {{host}}/uploads/podcast/news-2022-06-27/the-changelog-news-2022-06-27.mp3
+HTTP 308 # expect permanent redirect response
+[Asserts]
+header "Content-Length" == "0"
+header "Location" contains "/uploads/news/1/changelog-news-1.mp3"
```

Comment:
I recommend adding an assertion that you get `Permanent Redirect` as the HTTP status message. Not having it won't break anything, but it's nice for debugging.

Reply:
Yes, that's the role of the `HTTP 308` right after the `GET` request: https://hurl.dev/docs/asserting-response.html

![image](https://github.com/user-attachments/assets/166fdad8-ebab-4afc-bfc3-be1ffe9667ea)

Reply:
> That's the role of the HTTP 308 right after the GET request

That's testing the status code (number), not the status message (string). `synth(308, "Waffles Are Tasty")` would return a `308 Waffles Are Tasty`.